aws_lambda_builders/workflows/rust_cargo/workflow.py

```diff
@@ -0,0 +1,37 @@
+"""
+Rust Cargo Workflow
+"""
+import platform
+from aws_lambda_builders.path_resolver import PathResolver
+from aws_lambda_builders.workflow import BaseWorkflow, Capability
+from .actions import BuildAction, CopyAndRenameAction
+
+
+class RustCargoWorkflow(BaseWorkflow):
+
+    NAME = "RustCargoBuilder"
+
+    CAPABILITY = Capability(language="rust", dependency_manager="cargo", application_framework=None)
+
+    SUPPORTED_MANIFESTS = ["Cargo.toml"]
+
+    def __init__(self, source_dir, artifacts_dir, scratch_dir, manifest_path, runtime=None, mode=None, **kwargs):
+        super(RustCargoWorkflow, self).__init__(
+            source_dir, artifacts_dir, scratch_dir, manifest_path, runtime=runtime, **kwargs
+        )
+
+        # we utilize the handler identifier to
+        # select the binary to build
+        options = kwargs.get("options") or {}
+        handler = options.get("artifact_executable_name", None)
```

Comment:
I'm reusing [this interface](https://github.com/awslabs/aws-sam-cli/blob/39add9acd0c7532129a6543134919eb6ae9df4f3/samcli/lib/build/app_builder.py#L291) used also by the go runtime to pass along a reference to a handler identifier. I didn't want to reinvent another interface